docs/installation-and-operation/information-collection.md

```diff
@@ -1,104 +1,29 @@
 ---
-title: About the information we collect
+title: About the anonymous usage data we collect
 redirect_from:
   - /docs/latest/information-collection
 ---
 
-# About the information we collect
+# About the anonymous usage data we collect
 
-Metabase uses Google Analytics and Snowplow to collect anonymous usage information from the installed servers that enable this feature. Below is a representative list of the events we have instrumented, as well as the information we collect about the user performing the action and the instance being used.
+If you've opted in to provide us with anonymous usage data (thank you!), Metabase will phone home some data collected via Google Analytics and Snowplow.
 
-We also will phone home some anonymized metrics from the metabase application server on a nightly basis. Note that we don't collect any usernames, any emails, the server IP, database details of any kind or any personally identifiable information in this process.
+**We only collect Metabase data; we don't collect any of your data.** We don't collect any usernames, emails, server IPs, database details of any kind, or any personally identifiable information.
 
-While this list of anonymous information we collect might seem long, it’s useful to compare this to other alternatives. With a typical SaaS platform, not only will this information be collected, but it will also be accompanied by information about your data, how often it is accessed, the specific queries that you use, specific numbers of records all tied to your company and current plan.
+The data helps us understand how people are actually using Metabase, which in turn helps us prioritize what to work on next.
 
-We collect this information to improve your experience and the quality of Metabase. We use the information you voluntarily share to understand how our users are actually using our product, what kinds of features to prioritize, and how many items to target in our design process. For example, without knowing what the distribution of the number of accounts on each instance in our install base is, we can't know whether the tools we provide can't scale until someone complains. And even then, we only hear complaints and not the people who are quietly happily using us. We're striving to create the best product possible.
+## Opting out of anonymous usage data collection
 
-If you prefer not to provide us with this anonymous usage data, you can always go to your instance’s admin section and toggle off the option for `Anonymous Tracking`.
+To opt out of providing us with your anonymous usage data:
```

Comment:
We need to clarify that this is for self-hosted instances, [we're removing that opt in from Cloud](https://www.notion.so/metabase/Reduce-the-number-of-choices-in-onboarding-flow-f74849658dce4cceb9a9fd33b243f820?pvs=4#70f76b4db3a4410d920cfef1bb404236).